transform.adoc

```diff
@@ -37,7 +37,7 @@ The transform object has the following properties:
 
 - The jsonPath selector is an identifier of the location of the _input_ path. All paths should start with the root object `$`. The exception here is for an Array of objects, where the relative `@` operator should be used on the leaf objects to signify the location is relative to the parent transform (more on this below)
 
-- The xPath selector is an identifier of the location of the _input_ path. All paths should start with the root object `/`. The exception here is for an Array of objects, where `//` can be used to select a node and all of its descendants. The xPath inside the array can then be relative to this node.
+- The xPath selector is an identifier of the location of the _input_ path. All paths should start with the root object `/`. The exception here is for an Array of objects, where `//` can be used to select a node and all of its descendants. The xPath inside the array can then be relative to this node. An object can also use an xPath that will result in the selection of a node in which the fields inside the object will be able to be referenced relative the base object's node.
```

Comment:
Maybe rather than `All paths should start with the root object` it is time to say something like `Many paths will start with the root object '/' but items in arrays and objects can be relative. . . .`